config/opensearch_dashboards.yml

```diff
@@ -105,3 +105,38 @@
 # Specifies locale to be used for all localizable strings, dates and number formats.
 # Supported languages are the following: English - en , by default , Chinese - zh-CN .
 #i18n.locale: "en"
+
+# Set the allowlist to check input graphite Url. Allowlist is the default check list.
+#vis_type_timeline.graphiteAllowedUrls: ['https://www.hostedgraphite.com/UID/ACCESS_KEY/graphite']
+
+# Set the blocklist to check input graphite Url. Blocklist is an IP list. 
+# Below is an example for reference 
+# vis_type_timeline.graphiteBlockedIPs: [
+#  //Loopback
```

Comment:
NIT: do these need the //

Reply:
I think for this part, I am trying to provide customer a relative complete list of blocklist. So customer can remove `#` and use directly. For example, AWS can add some internal ips and no need to comment any further.